extensions/amp-story/0.1/related-articles.js

```diff
@@ -56,11 +56,20 @@ function buildArticleFromJson_(articleJson) {
 
   user().assert(isProtocolValid(articleJson['url']),
```

Comment:
Nit: maybe `const articleUrl = dev().assert(articleJson['url'])` before this line